src/tests/docker-compose.yml

```diff
@@ -59,11 +59,17 @@ services:
       - PG_HOST=timescale
       - PG_PASS=*
 
-
   crate:
     image: crate:${CRATE_VERSION:-4.5.1}
-    command: crate -Cauth.host_based.enabled=false
-      -Ccluster.name=democluster -Chttp.cors.enabled=true -Chttp.cors.allow-origin="*"
+    command: crate -Cauth.host_based.enabled=true
```

Comment:
this is the reason why bc test are failing, ql 0.7.5 of course does not support auth